### ISSUE TITLE
Pasting long text causes clink to crash

### ISSUE THREAD

**Issue by jack1142** [chrisant996/clink#115]
I tried to paste this:

<details>
<summary>Details</summary>

```
git rm redbot/cogs/audio/apis/api_utils.py redbot/cogs/audio/apis/global_db.py redbot/cogs/audio/apis/interface.py redbot/cogs/audio/apis/local_db.py redbot/cogs/audio/apis/locales/af-ZA.po redbot/cogs/audio/apis/locales/ar-SA.po redbot/cogs/audio/apis/locales/bg-BG.po redbot/cogs/audio/apis/locales/ca-ES.po redbot/cogs/audio/apis/locales/cs-CZ.po redbot/cogs/audio/apis/locales/da-DK.po redbot/cogs/audio/apis/locales/de-DE.po redbot/cogs/audio/apis/locales/el-GR.po redbot/cogs/audio/apis/locales/es-ES.po redbot/cogs/audio/apis/locales/fi-FI.po redbot/cogs/audio/apis/locales/fr-FR.po redbot/cogs/audio/apis/locales/he-IL.po redbot/cogs/audio/apis/locales/hu-HU.po redbot/cogs/audio/apis/locales/id-ID.po redbot/cogs/audio/apis/locales/it-IT.po redbot/cogs/audio/apis/locales/ja-JP.po redbot/cogs/audio/apis/locales/ko-KR.po redbot/cogs/audio/apis/locales/nl-NL.po redbot/cogs/audio/apis/locales/pl-PL.po redbot/cogs/audio/apis/locales/pt-BR.po redbot/cogs/audio/apis/locales/pt-PT.po redbot/cogs/audio/apis/locales/ro-RO.po redbot/cogs/audio/apis/locales/ru-RU.po redbot/cogs/audio/apis/locales/sk-SK.po redbot/cogs/audio/apis/locales/sr-SP.po redbot/cogs/audio/apis/locales/sv-SE.po redbot/cogs/audio/apis/locales/tr-TR.po redbot/cogs/audio/apis/locales/uk-UA.po redbot/cogs/audio/apis/locales/vi-VN.po redbot/cogs/audio/apis/locales/zh-CN.po redbot/cogs/audio/apis/locales/zh-HK.po redbot/cogs/audio/apis/locales/zh-TW.po redbot/cogs/audio/apis/playlist_interface.py redbot/cogs/audio/apis/playlist_wrapper.py redbot/cogs/audio/apis/spotify.py redbot/cogs/audio/apis/youtube.py redbot/cogs/audio/audio_dataclasses.py redbot/cogs/audio/audio_logging.py redbot/cogs/audio/converters.py redbot/cogs/audio/core/__init__.py redbot/cogs/audio/core/abc.py redbot/cogs/audio/core/cog_utils.py redbot/cogs/audio/core/commands/audioset.py redbot/cogs/audio/core/commands/controller.py redbot/cogs/audio/core/commands/equalizer.py redbot/cogs/audio/core/commands/llset.py redbot/cogs/audio/core/commands/locales/af-ZA.po redbot/cogs/audio/core/commands/locales/ar-SA.po redbot/cogs/audio/core/commands/locales/bg-BG.po redbot/cogs/audio/core/commands/locales/ca-ES.po redbot/cogs/audio/core/commands/locales/cs-CZ.po redbot/cogs/audio/core/commands/locales/da-DK.po redbot/cogs/audio/core/commands/locales/de-DE.po redbot/cogs/audio/core/commands/locales/el-GR.po redbot/cogs/audio/core/commands/locales/es-ES.po redbot/cogs/audio/core/commands/locales/fi-FI.po redbot/cogs/audio/core/commands/locales/fr-FR.po redbot/cogs/audio/core/commands/locales/he-IL.po redbot/cogs/audio/core/commands/locales/hu-HU.po redbot/cogs/audio/core/commands/locales/id-ID.po redbot/cogs/audio/core/commands/locales/it-IT.po redbot/cogs/audio/core/commands/locales/ja-JP.po redbot/cogs/audio/core/commands/locales/ko-KR.po redbot/cogs/audio/core/commands/locales/nl-NL.po redbot/cogs/audio/core/commands/locales/pl-PL.po redbot/cogs/audio/core/commands/locales/pt-BR.po redbot/cogs/audio/core/commands/locales/pt-PT.po redbot/cogs/audio/core/commands/locales/ro-RO.po redbot/cogs/audio/core/commands/locales/ru-RU.po redbot/cogs/audio/core/commands/locales/sk-SK.po redbot/cogs/audio/core/commands/locales/sr-SP.po redbot/cogs/audio/core/commands/locales/sv-SE.po redbot/cogs/audio/core/commands/locales/tr-TR.po redbot/cogs/audio/core/commands/locales/uk-UA.po redbot/cogs/audio/core/commands/locales/vi-VN.po redbot/cogs/audio/core/commands/locales/zh-CN.po redbot/cogs/audio/core/commands/locales/zh-HK.po redbot/cogs/audio/core/commands/locales/zh-TW.po redbot/cogs/audio/core/commands/localtracks.py redbot/cogs/audio/core/commands/miscellaneous.py redbot/cogs/audio/core/commands/player.py redbot/cogs/audio/core/commands/playlists.py redbot/cogs/audio/core/commands/queue.py redbot/cogs/audio/core/events/cog.py redbot/cogs/audio/core/events/dpy.py redbot/cogs/audio/core/events/lavalink.py redbot/cogs/audio/core/events/locales/af-ZA.po redbot/cogs/audio/core/events/locales/ar-SA.po redbot/cogs/audio/core/events/locales/bg-BG.po redbot/cogs/audio/core/events/locales/ca-ES.po redbot/cogs/audio/core/events/locales/cs-CZ.po redbot/cogs/audio/core/events/locales/da-DK.po redbot/cogs/audio/core/events/locales/de-DE.po redbot/cogs/audio/core/events/locales/el-GR.po redbot/cogs/audio/core/events/locales/es-ES.po redbot/cogs/audio/core/events/locales/fi-FI.po redbot/cogs/audio/core/events/locales/fr-FR.po redbot/cogs/audio/core/events/locales/he-IL.po redbot/cogs/audio/core/events/locales/hu-HU.po redbot/cogs/audio/core/events/locales/id-ID.po redbot/cogs/audio/core/events/locales/it-IT.po redbot/cogs/audio/core/events/locales/ja-JP.po redbot/cogs/audio/core/events/locales/ko-KR.po redbot/cogs/audio/core/events/locales/nl-NL.po redbot/cogs/audio/core/events/locales/pl-PL.po redbot/cogs/audio/core/events/locales/pt-BR.po redbot/cogs/audio/core/events/locales/pt-PT.po redbot/cogs/audio/core/events/locales/ro-RO.po redbot/cogs/audio/core/events/locales/ru-RU.po redbot/cogs/audio/core/events/locales/sk-SK.po redbot/cogs/audio/core/events/locales/sr-SP.po redbot/cogs/audio/core/events/locales/sv-SE.po redbot/cogs/audio/core/events/locales/tr-TR.po redbot/cogs/audio/core/events/locales/uk-UA.po redbot/cogs/audio/core/events/locales/vi-VN.po redbot/cogs/audio/core/events/locales/zh-CN.po redbot/cogs/audio/core/events/locales/zh-HK.po redbot/cogs/audio/core/events/locales/zh-TW.po redbot/cogs/audio/core/events/red.py redbot/cogs/audio/core/locales/af-ZA.po redbot/cogs/audio/core/locales/ar-SA.po redbot/cogs/audio/core/locales/bg-BG.po redbot/cogs/audio/core/locales/ca-ES.po redbot/cogs/audio/core/locales/cs-CZ.po redbot/cogs/audio/core/locales/da-DK.po redbot/cogs/audio/core/locales/de-DE.po redbot/cogs/audio/core/locales/el-GR.po redbot/cogs/audio/core/locales/es-ES.po redbot/cogs/audio/core/locales/fi-FI.po redbot/cogs/audio/core/locales/fr-FR.po redbot/cogs/audio/core/locales/he-IL.po redbot/cogs/audio/core/locales/hu-HU.po redbot/cogs/audio/core/locales/id-ID.po redbot/cogs/audio/core/locales/it-IT.po redbot/cogs/audio/core/locales/ja-JP.po redbot/cogs/audio/core/locales/ko-KR.po redbot/cogs/audio/core/locales/nl-NL.po redbot/cogs/audio/core/locales/pl-PL.po redbot/cogs/audio/core/locales/pt-BR.po redbot/cogs/audio/core/locales/pt-PT.po redbot/cogs/audio/core/locales/ro-RO.po redbot/cogs/audio/core/locales/ru-RU.po redbot/cogs/audio/core/locales/sk-SK.po redbot/cogs/audio/core/locales/sr-SP.po redbot/cogs/audio/core/locales/sv-SE.po redbot/cogs/audio/core/locales/tr-TR.po redbot/cogs/audio/core/locales/uk-UA.po redbot/cogs/audio/core/locales/vi-VN.po redbot/cogs/audio/core/locales/zh-CN.po redbot/cogs/audio/core/locales/zh-HK.po redbot/cogs/audio/core/locales/zh-TW.po redbot/cogs/audio/core/tasks/lavalink.py redbot/cogs/audio/core/tasks/locales/af-ZA.po redbot/cogs/audio/core/tasks/locales/ar-SA.po redbot/cogs/audio/core/tasks/locales/bg-BG.po redbot/cogs/audio/core/tasks/locales/ca-ES.po redbot/cogs/audio/core/tasks/locales/cs-CZ.po redbot/cogs/audio/core/tasks/locales/da-DK.po redbot/cogs/audio/core/tasks/locales/de-DE.po redbot/cogs/audio/core/tasks/locales/el-GR.po redbot/cogs/audio/core/tasks/locales/es-ES.po redbot/cogs/audio/core/tasks/locales/fi-FI.po redbot/cogs/audio/core/tasks/locales/fr-FR.po redbot/cogs/audio/core/tasks/locales/he-IL.po redbot/cogs/audio/core/tasks/locales/hu-HU.po redbot/cogs/audio/core/tasks/locales/id-ID.po redbot/cogs/audio/core/tasks/locales/it-IT.po redbot/cogs/audio/core/tasks/locales/ja-JP.po redbot/cogs/audio/core/tasks/locales/ko-KR.po redbot/cogs/audio/core/tasks/locales/nl-NL.po redbot/cogs/audio/core/tasks/locales/pl-PL.po redbot/cogs/audio/core/tasks/locales/pt-BR.po redbot/cogs/audio/core/tasks/locales/pt-PT.po redbot/cogs/audio/core/tasks/locales/ro-RO.po redbot/cogs/audio/core/tasks/locales/ru-RU.po redbot/cogs/audio/core/tasks/locales/sk-SK.po redbot/cogs/audio/core/tasks/locales/sr-SP.po redbot/cogs/audio/core/tasks/locales/sv-SE.po redbot/cogs/audio/core/tasks/locales/tr-TR.po redbot/cogs/audio/core/tasks/locales/uk-UA.po redbot/cogs/audio/core/tasks/locales/vi-VN.po redbot/cogs/audio/core/tasks/locales/zh-CN.po redbot/cogs/audio/core/tasks/locales/zh-HK.po redbot/cogs/audio/core/tasks/locales/zh-TW.po redbot/cogs/audio/core/tasks/player.py redbot/cogs/audio/core/tasks/startup.py redbot/cogs/audio/core/utilities/__init__.py redbot/cogs/audio/core/utilities/equalizer.py redbot/cogs/audio/core/utilities/formatting.py redbot/cogs/audio/core/utilities/local_tracks.py redbot/cogs/audio/core/utilities/locales/af-ZA.po redbot/cogs/audio/core/utilities/locales/ar-SA.po redbot/cogs/audio/core/utilities/locales/bg-BG.po redbot/cogs/audio/core/utilities/locales/ca-ES.po redbot/cogs/audio/core/utilities/locales/cs-CZ.po redbot/cogs/audio/core/utilities/locales/da-DK.po redbot/cogs/audio/core/utilities/locales/de-DE.po redbot/cogs/audio/core/utilities/locales/el-GR.po redbot/cogs/audio/core/utilities/locales/es-ES.po redbot/cogs/audio/core/utilities/locales/fi-FI.po redbot/cogs/audio/core/utilities/locales/fr-FR.po redbot/cogs/audio/core/utilities/locales/he-IL.po redbot/cogs/audio/core/utilities/locales/hu-HU.po redbot/cogs/audio/core/utilities/locales/id-ID.po redbot/cogs/audio/core/utilities/locales/it-IT.po redbot/cogs/audio/core/utilities/locales/ja-JP.po redbot/cogs/audio/core/utilities/locales/ko-KR.po redbot/cogs/audio/core/utilities/locales/nl-NL.po redbot/cogs/audio/core/utilities/locales/pl-PL.po redbot/cogs/audio/core/utilities/locales/pt-BR.po redbot/cogs/audio/core/utilities/locales/pt-PT.po redbot/cogs/audio/core/utilities/locales/ro-RO.po redbot/cogs/audio/core/utilities/locales/ru-RU.po redbot/cogs/audio/core/utilities/locales/sk-SK.po redbot/cogs/audio/core/utilities/locales/sr-SP.po redbot/cogs/audio/core/utilities/locales/sv-SE.po redbot/cogs/audio/core/utilities/locales/tr-TR.po redbot/cogs/audio/core/utilities/locales/uk-UA.po redbot/cogs/audio/core/utilities/locales/vi-VN.po redbot/cogs/audio/core/utilities/locales/zh-CN.po redbot/cogs/audio/core/utilities/locales/zh-HK.po redbot/cogs/audio/core/utilities/locales/zh-TW.po redbot/cogs/audio/core/utilities/menus/locales/af-ZA.po redbot/cogs/audio/core/utilities/menus/locales/ar-SA.po redbot/cogs/audio/core/utilities/menus/locales/bg-BG.po redbot/cogs/audio/core/utilities/menus/locales/ca-ES.po redbot/cogs/audio/core/utilities/menus/locales/cs-CZ.po redbot/cogs/audio/core/utilities/menus/locales/da-DK.po redbot/cogs/audio/core/utilities/menus/locales/de-DE.po redbot/cogs/audio/core/utilities/menus/locales/el-GR.po redbot/cogs/audio/core/utilities/menus/locales/es-ES.po redbot/cogs/audio/core/utilities/menus/locales/fi-FI.po redbot/cogs/audio/core/utilities/menus/locales/fr-FR.po redbot/cogs/audio/core/utilities/menus/locales/he-IL.po redbot/cogs/audio/core/utilities/menus/locales/hu-HU.po redbot/cogs/audio/core/utilities/menus/locales/id-ID.po redbot/cogs/audio/core/utilities/menus/locales/it-IT.po redbot/cogs/audio/core/utilities/menus/locales/ja-JP.po redbot/cogs/audio/core/utilities/menus/locales/ko-KR.po redbot/cogs/audio/core/utilities/menus/locales/nl-NL.po redbot/cogs/audio/core/utilities/menus/locales/pl-PL.po redbot/cogs/audio/core/utilities/menus/locales/pt-BR.po redbot/cogs/audio/core/utilities/menus/locales/pt-PT.po redbot/cogs/audio/core/utilities/menus/locales/ro-RO.po redbot/cogs/audio/core/utilities/menus/locales/ru-RU.po redbot/cogs/audio/core/utilities/menus/locales/sk-SK.po redbot/cogs/audio/core/utilities/menus/locales/sr-SP.po redbot/cogs/audio/core/utilities/menus/locales/sv-SE.po redbot/cogs/audio/core/utilities/menus/locales/tr-TR.po redbot/cogs/audio/core/utilities/menus/locales/uk-UA.po redbot/cogs/audio/core/utilities/menus/locales/vi-VN.po redbot/cogs/audio/core/utilities/menus/locales/zh-CN.po redbot/cogs/audio/core/utilities/menus/locales/zh-HK.po redbot/cogs/audio/core/utilities/menus/locales/zh-TW.po redbot/cogs/audio/core/utilities/miscellaneous.py redbot/cogs/audio/core/utilities/player.py redbot/cogs/audio/core/utilities/playlists.py redbot/cogs/audio/core/utilities/queue.py redbot/cogs/audio/core/utilities/validation.py redbot/cogs/audio/data/application.yml redbot/cogs/audio/equalizer.py redbot/cogs/audio/errors.py redbot/cogs/audio/locales/af-ZA.po redbot/cogs/audio/locales/ar-SA.po redbot/cogs/audio/locales/bg-BG.po redbot/cogs/audio/locales/ca-ES.po redbot/cogs/audio/locales/cs-CZ.po redbot/cogs/audio/locales/da-DK.po redbot/cogs/audio/locales/de-DE.po redbot/cogs/audio/locales/el-GR.po redbot/cogs/audio/locales/es-ES.po redbot/cogs/audio/locales/fi-FI.po redbot/cogs/audio/locales/fr-FR.po redbot/cogs/audio/locales/he-IL.po redbot/cogs/audio/locales/hu-HU.po redbot/cogs/audio/locales/id-ID.po redbot/cogs/audio/locales/it-IT.po redbot/cogs/audio/locales/ja-JP.po redbot/cogs/audio/locales/ko-KR.po redbot/cogs/audio/locales/nl-NL.po redbot/cogs/audio/locales/pl-PL.po redbot/cogs/audio/locales/pt-BR.po redbot/cogs/audio/locales/pt-PT.po redbot/cogs/audio/locales/ro-RO.po redbot/cogs/audio/locales/ru-RU.po redbot/cogs/audio/locales/sk-SK.po redbot/cogs/audio/locales/sr-SP.po redbot/cogs/audio/locales/sv-SE.po redbot/cogs/audio/locales/tr-TR.po redbot/cogs/audio/locales/uk-UA.po redbot/cogs/audio/locales/vi-VN.po redbot/cogs/audio/locales/zh-CN.po redbot/cogs/audio/locales/zh-HK.po redbot/cogs/audio/locales/zh-TW.po redbot/cogs/audio/manager.py redbot/cogs/audio/sql_statements.py
```

</details>

Which resulted in this:
```
!!! CLINK'S CRASHED!
!!!
!!! Writing core dump
!!! C:\tools\cmdermini\config\clink.dmp
!!! ...ok
!!!
!!! Clink: 0x00007FFC384A0000
!!! Host: 0x00007FF71A2C0000
!!! Guru: 0xc0000005 addr=0x0000000000000000 (1)
!!!
!!! Backtrace:
!!! 0x00007FFC384D2A1A
!!! 0x00007FFC384BE348
!!! 0x00007FFC3852C408
!!! 0x00007FFC38520536
!!! 0x00007FFC385207B6
!!! 0x00007FFC38538BE7
!!! 0x00007FFC384C45DF
!!! 0x00007FFC384BDB0C
!!! 0x00007FFC384BD286
!!! 0x00007FFC384BCFF6
!!! 0x00007FFC384B3ED8
!!! 0x00007FFC384B0FAA
!!! 0x00007FFC384B120F
!!! 0x00007FF71A2E8027
!!! 0x00007FF71A2DD31B
!!! 0x00007FF71A2CF438
!!! 0x00007FF71A2CE626
!!! 0x00007FF71A2CE073
!!! 0x00007FF71A2DEBA6
!!! 0x00007FF71A2D8ECD
!!! 0x00007FFCAB687034
!!! 0x00007FFCAC202651
```

**Comment by chrisant996** [comment]
Thanks, I'll take a look.

One thing to be aware of:  the text being pasted is longer than the maximum CMD command line.  So while it's possible to fix the crash, it isn't possible to make executing the command work as expected, because CMD doesn't support it.

**Comment by jack1142** [comment]
> So while it's possible to fix the crash, it isn't possible to make executing the command work as expected, because CMD doesn't support it.

Yup, I know that - CMD just cuts past the supported length.

However, I am able to replicate the same problem with clipboard contents that don't exceed the maximum length too (8165 characters while the max is 8191):

<details>
<summary>Details</summary>

```
git rm redbot/cogs/audio/apis/api_utils.py redbot/cogs/audio/apis/global_db.py redbot/cogs/audio/apis/interface.py redbot/cogs/audio/apis/local_db.py redbot/cogs/audio/apis/locales/af-ZA.po redbot/cogs/audio/apis/locales/ar-SA.po redbot/cogs/audio/apis/locales/bg-BG.po redbot/cogs/audio/apis/locales/ca-ES.po redbot/cogs/audio/apis/locales/cs-CZ.po redbot/cogs/audio/apis/locales/da-DK.po redbot/cogs/audio/apis/locales/de-DE.po redbot/cogs/audio/apis/locales/el-GR.po redbot/cogs/audio/apis/locales/es-ES.po redbot/cogs/audio/apis/locales/fi-FI.po redbot/cogs/audio/apis/locales/fr-FR.po redbot/cogs/audio/apis/locales/he-IL.po redbot/cogs/audio/apis/locales/hu-HU.po redbot/cogs/audio/apis/locales/id-ID.po redbot/cogs/audio/apis/locales/it-IT.po redbot/cogs/audio/apis/locales/ja-JP.po redbot/cogs/audio/apis/locales/ko-KR.po redbot/cogs/audio/apis/locales/nl-NL.po redbot/cogs/audio/apis/locales/pl-PL.po redbot/cogs/audio/apis/locales/pt-BR.po redbot/cogs/audio/apis/locales/pt-PT.po redbot/cogs/audio/apis/locales/ro-RO.po redbot/cogs/audio/apis/locales/ru-RU.po redbot/cogs/audio/apis/locales/sk-SK.po redbot/cogs/audio/apis/locales/sr-SP.po redbot/cogs/audio/apis/locales/sv-SE.po redbot/cogs/audio/apis/locales/tr-TR.po redbot/cogs/audio/apis/locales/uk-UA.po redbot/cogs/audio/apis/locales/vi-VN.po redbot/cogs/audio/apis/locales/zh-CN.po redbot/cogs/audio/apis/locales/zh-HK.po redbot/cogs/audio/apis/locales/zh-TW.po redbot/cogs/audio/apis/playlist_interface.py redbot/cogs/audio/apis/playlist_wrapper.py redbot/cogs/audio/apis/spotify.py redbot/cogs/audio/apis/youtube.py redbot/cogs/audio/audio_dataclasses.py redbot/cogs/audio/audio_logging.py redbot/cogs/audio/converters.py redbot/cogs/audio/core/__init__.py redbot/cogs/audio/core/abc.py redbot/cogs/audio/core/cog_utils.py redbot/cogs/audio/core/commands/audioset.py redbot/cogs/audio/core/commands/controller.py redbot/cogs/audio/core/commands/equalizer.py redbot/cogs/audio/core/commands/llset.py redbot/cogs/audio/core/commands/locales/af-ZA.po redbot/cogs/audio/core/commands/locales/ar-SA.po redbot/cogs/audio/core/commands/locales/bg-BG.po redbot/cogs/audio/core/commands/locales/ca-ES.po redbot/cogs/audio/core/commands/locales/cs-CZ.po redbot/cogs/audio/core/commands/locales/da-DK.po redbot/cogs/audio/core/commands/locales/de-DE.po redbot/cogs/audio/core/commands/locales/el-GR.po redbot/cogs/audio/core/commands/locales/es-ES.po redbot/cogs/audio/core/commands/locales/fi-FI.po redbot/cogs/audio/core/commands/locales/fr-FR.po redbot/cogs/audio/core/commands/locales/he-IL.po redbot/cogs/audio/core/commands/locales/hu-HU.po redbot/cogs/audio/core/commands/locales/id-ID.po redbot/cogs/audio/core/commands/locales/it-IT.po redbot/cogs/audio/core/commands/locales/ja-JP.po redbot/cogs/audio/core/commands/locales/ko-KR.po redbot/cogs/audio/core/commands/locales/nl-NL.po redbot/cogs/audio/core/commands/locales/pl-PL.po redbot/cogs/audio/core/commands/locales/pt-BR.po redbot/cogs/audio/core/commands/locales/pt-PT.po redbot/cogs/audio/core/commands/locales/ro-RO.po redbot/cogs/audio/core/commands/locales/ru-RU.po redbot/cogs/audio/core/commands/locales/sk-SK.po redbot/cogs/audio/core/commands/locales/sr-SP.po redbot/cogs/audio/core/commands/locales/sv-SE.po redbot/cogs/audio/core/commands/locales/tr-TR.po redbot/cogs/audio/core/commands/locales/uk-UA.po redbot/cogs/audio/core/commands/locales/vi-VN.po redbot/cogs/audio/core/commands/locales/zh-CN.po redbot/cogs/audio/core/commands/locales/zh-HK.po redbot/cogs/audio/core/commands/locales/zh-TW.po redbot/cogs/audio/core/commands/localtracks.py redbot/cogs/audio/core/commands/miscellaneous.py redbot/cogs/audio/core/commands/player.py redbot/cogs/audio/core/commands/playlists.py redbot/cogs/audio/core/commands/queue.py redbot/cogs/audio/core/events/cog.py redbot/cogs/audio/core/events/dpy.py redbot/cogs/audio/core/events/lavalink.py redbot/cogs/audio/core/events/locales/af-ZA.po redbot/cogs/audio/core/events/locales/ar-SA.po redbot/cogs/audio/core/events/locales/bg-BG.po redbot/cogs/audio/core/events/locales/ca-ES.po redbot/cogs/audio/core/events/locales/cs-CZ.po redbot/cogs/audio/core/events/locales/da-DK.po redbot/cogs/audio/core/events/locales/de-DE.po redbot/cogs/audio/core/events/locales/el-GR.po redbot/cogs/audio/core/events/locales/es-ES.po redbot/cogs/audio/core/events/locales/fi-FI.po redbot/cogs/audio/core/events/locales/fr-FR.po redbot/cogs/audio/core/events/locales/he-IL.po redbot/cogs/audio/core/events/locales/hu-HU.po redbot/cogs/audio/core/events/locales/id-ID.po redbot/cogs/audio/core/events/locales/it-IT.po redbot/cogs/audio/core/events/locales/ja-JP.po redbot/cogs/audio/core/events/locales/ko-KR.po redbot/cogs/audio/core/events/locales/nl-NL.po redbot/cogs/audio/core/events/locales/pl-PL.po redbot/cogs/audio/core/events/locales/pt-BR.po redbot/cogs/audio/core/events/locales/pt-PT.po redbot/cogs/audio/core/events/locales/ro-RO.po redbot/cogs/audio/core/events/locales/ru-RU.po redbot/cogs/audio/core/events/locales/sk-SK.po redbot/cogs/audio/core/events/locales/sr-SP.po redbot/cogs/audio/core/events/locales/sv-SE.po redbot/cogs/audio/core/events/locales/tr-TR.po redbot/cogs/audio/core/events/locales/uk-UA.po redbot/cogs/audio/core/events/locales/vi-VN.po redbot/cogs/audio/core/events/locales/zh-CN.po redbot/cogs/audio/core/events/locales/zh-HK.po redbot/cogs/audio/core/events/locales/zh-TW.po redbot/cogs/audio/core/events/red.py redbot/cogs/audio/core/locales/af-ZA.po redbot/cogs/audio/core/locales/ar-SA.po redbot/cogs/audio/core/locales/bg-BG.po redbot/cogs/audio/core/locales/ca-ES.po redbot/cogs/audio/core/locales/cs-CZ.po redbot/cogs/audio/core/locales/da-DK.po redbot/cogs/audio/core/locales/de-DE.po redbot/cogs/audio/core/locales/el-GR.po redbot/cogs/audio/core/locales/es-ES.po redbot/cogs/audio/core/locales/fi-FI.po redbot/cogs/audio/core/locales/fr-FR.po redbot/cogs/audio/core/locales/he-IL.po redbot/cogs/audio/core/locales/hu-HU.po redbot/cogs/audio/core/locales/id-ID.po redbot/cogs/audio/core/locales/it-IT.po redbot/cogs/audio/core/locales/ja-JP.po redbot/cogs/audio/core/locales/ko-KR.po redbot/cogs/audio/core/locales/nl-NL.po redbot/cogs/audio/core/locales/pl-PL.po redbot/cogs/audio/core/locales/pt-BR.po redbot/cogs/audio/core/locales/pt-PT.po redbot/cogs/audio/core/locales/ro-RO.po redbot/cogs/audio/core/locales/ru-RU.po redbot/cogs/audio/core/locales/sk-SK.po redbot/cogs/audio/core/locales/sr-SP.po redbot/cogs/audio/core/locales/sv-SE.po redbot/cogs/audio/core/locales/tr-TR.po redbot/cogs/audio/core/locales/uk-UA.po redbot/cogs/audio/core/locales/vi-VN.po redbot/cogs/audio/core/locales/zh-CN.po redbot/cogs/audio/core/locales/zh-HK.po redbot/cogs/audio/core/locales/zh-TW.po redbot/cogs/audio/core/tasks/lavalink.py redbot/cogs/audio/core/tasks/locales/af-ZA.po redbot/cogs/audio/core/tasks/locales/ar-SA.po redbot/cogs/audio/core/tasks/locales/bg-BG.po redbot/cogs/audio/core/tasks/locales/ca-ES.po redbot/cogs/audio/core/tasks/locales/cs-CZ.po redbot/cogs/audio/core/tasks/locales/da-DK.po redbot/cogs/audio/core/tasks/locales/de-DE.po redbot/cogs/audio/core/tasks/locales/el-GR.po redbot/cogs/audio/core/tasks/locales/es-ES.po redbot/cogs/audio/core/tasks/locales/fi-FI.po redbot/cogs/audio/core/tasks/locales/fr-FR.po redbot/cogs/audio/core/tasks/locales/he-IL.po redbot/cogs/audio/core/tasks/locales/hu-HU.po redbot/cogs/audio/core/tasks/locales/id-ID.po redbot/cogs/audio/core/tasks/locales/it-IT.po redbot/cogs/audio/core/tasks/locales/ja-JP.po redbot/cogs/audio/core/tasks/locales/ko-KR.po redbot/cogs/audio/core/tasks/locales/nl-NL.po redbot/cogs/audio/core/tasks/locales/pl-PL.po redbot/cogs/audio/core/tasks/locales/pt-BR.po redbot/cogs/audio/core/tasks/locales/pt-PT.po redbot/cogs/audio/core/tasks/locales/ro-RO.po redbot/cogs/audio/core/tasks/locales/ru-RU.po redbot/cogs/audio/core/tasks/locales/sk-SK.po redbot/cogs/audio/core/tasks/locales/sr-SP.po redbot/cogs/audio/core/tasks/locales/sv-SE.po redbot/cogs/audio/core/tasks/locales/tr-TR.po redbot/cogs/audio/core/tasks/locales/uk-UA.po redbot/cogs/audio/core/tasks/locales/vi-VN.po redbot/cogs/audio/core/tasks/locales/zh-CN.po redbot/cogs/audio/core/tasks/locales/zh-HK.po 
```

</details>

**Comment by chrisant996** [comment]
The crash doesn't seem to occur in released versions of Clink.
Cmder is still using a pre-release version of Clink (v1.1.45).
Can you try copying Clink v1.2 or higher into the `Cmder\vendor\clink` directory?

**Comment by jack1142** [comment]
Looks like you're right, it doesn't occur on the latest Clink version. Sorry for the trouble 😬

**Comment by chrisant996** [comment]
Was fixed by a48a874a515331744feeed1a17d59ff84cea3474 on Apr 3.
When Cmder picks up a new build of Clink, the issue will go away there.

> Looks like you're right, it doesn't occur on the latest Clink version. Sorry for the trouble 😬

Not at all!  Thank you for reporting the issue!!
